R/format_value.R

```diff
@@ -119,14 +119,19 @@ check_aligns <- function(algn) {
 #' @export
 #' @return A formatting function which wraps and will apply the specified \code{printf} style format
 #'   string \code{format}.
-#' @seealso \code{\link[base]{sprintf}}
-#'
+#' @seealso \code{\link[base]{sprintf}} [round_fmt()] \code{link{list_formats}}
```

Comment:
@martincadek could you make it all [fnc()] please? Also can you go in the other functions and see that they have some good connection with this one? otherwise it remains difficult to reach

Reply:
I think there should also be a counterpart in {rtables} with links to these in some of the obscure description of parameters. All of this would make a nice extension of the current vignette on format precedence